### PR TITLE
Refactor memory limits page

### PR DESCRIPTION
**Pull request by wcmjunior**

### CODE REVIEW

File: docs/memlim.md [hunk 1]
```diff
@@ -4,40 +4,32 @@ Each of the individual services within the sidecar has a default
...
+page of our public docs for more details.
 
-## Environment Variables
+In order to change the memory limits for your sidecar container, override the
+default settings using the [environment variables](https://cyral.com/docs/sidecars/deployment/memory-limits#environment-variables)
+detailed in our public docs.
 
-The following environment variables can be set to override the default memory
-limits.
+## Setting memory limits via environment file
```
Comment (ptcar2009) [inline docs/memlim.md]: Maybe put the environment variable session under the environment file title? That's just a nit though

**Review by ptcar2009 — APPROVED** [review]
Just a small comment, otherwise everything seems fine

File: docs/memlim.md (hunk 1 above)
Comment (wcmjunior) [reply docs/memlim.md]: Just adjusted the whole text to make it clearer and avoid the idea that the example section for the environment file is the only way to do it.

**Review by ptcar2009 — APPROVED** [review]
Changes improved it a lot

**Pull request merged**